crates/zed/src/main.rs

```diff
@@ -191,12 +191,25 @@ fn main() {
         watch_file_types(fs.clone(), cx);
 
         languages.set_theme(cx.theme().clone());
+        cx.windows().iter_mut().for_each(|&mut w| {
```

Comment:
Can we expand `w` to `window` to make it clearer?

```suggestion
        cx.windows().iter_mut().for_each(|&mut window| {
```